### PR TITLE
321 replace placeholder

### PR DESCRIPTION
**Pull request by Gkal2000**

### CODE REVIEW

**Review by almazhankapan — APPROVED** [review]

File: front-end/src/components/HeaderLogo.css [hunk 1]
```diff
@@ -2,7 +2,12 @@
   display: block;
   margin-left: auto;
   margin-right: auto;
+<<<<<<< HEAD
```
Comment (LevBernstein) [inline front-end/src/components/HeaderLogo.css:5]: This looks like a merge artifact; it needs to be sorted out before we can merge. I get an error whenever I view a page.

File: front-end/src/components/HeaderLogo.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
-import logo from '../images/logo.png'
+import logo from '../images/newLogo.png'
```
Comment (LevBernstein) [inline front-end/src/components/HeaderLogo.js:3]: newLogo seems to be an old version of the new logo; i'd suggest sticking with logo.png

File: front-end/src/components/Header.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import 'react-sticky-header/styles.css'
 import StickyHeader from 'react-sticky-header'
 import './Header.css'
 import Hamburger from './Hamburger'
-import logo from '../images/logo.png'
+import logo from '../images/newLogo.png'
```
Comment (LevBernstein) [inline front-end/src/components/Header.js:6]: newLogo seems to be an old version of the new logo; i'd suggest sticking with logo.png

**Review by LevBernstein — CHANGES_REQUESTED** [review]

**Comment by almazhankapan** [comment]
@LevBernstein I believe this is an outdated pr (should be closed); I picked up the task

**Comment by LevBernstein** [comment]
@almazhankapan last standup, this was what mason had said he was working on; wasn't the task assigned to him?

**Comment by almazhankapan** [comment]
@LevBernstein yup,  i previously approved the pr, but then mason suggested that i pick up the task (as i am working on images and there was an issue with pr);

**Comment by LevBernstein** [comment]
got it. mason should be assigned a new task, then

**Pull request closed without merging**